virtual_rainforest/models/soil/carbon.py

```diff
@@ -76,22 +73,25 @@ def calculate_soil_carbon_updates(
         bulk_density,
         moist_temp_scalar,
         percent_clay,
+        constants,
     )
     microbial_uptake = calculate_microbial_carbon_uptake(
-        soil_c_pool_lmwc, soil_c_pool_microbe, moist_temp_scalar, soil_temp
+        soil_c_pool_lmwc, soil_c_pool_microbe, moist_temp_scalar, soil_temp, constants
```

Comment:
Do you import `constants` when you use more than one?

Reply:
I messed this up a bit, but the idea was that `SoilConsts` is only importing for type hinting. Then `constants` is passed down the function stack as an argument, starting with `SoilModel.update` which passes `self.constants` avoiding any need for constants to be imported. (if that make sense?)

Reply:
makes sense 👍